pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
> your example seems to have assigned completely different data to it than before.

When you create a repository, you should get back attributes about the repositories created. Same for reading and updating. I'm assuming that there's minor differences, if any, between the response to a POST and GET request.